packages/suite/src/components/wallet/TransactionItem/TransactionTarget/TransactionTarget.tsx

```diff
@@ -68,15 +70,12 @@ export const TokenTransfer = ({
                 />
             }
             amount={
-                isNft ? (
-                    <StyledFormattedNftAmount transfer={transfer} signValue={operation} />
-                ) : (
-                    <StyledFormattedCryptoAmount
-                        value={formatAmount(transfer.amount, transfer.decimals)}
-                        symbol={transfer.symbol}
-                        signValue={operation}
-                    />
-                )
+                <AmountComponent
```

Comment:
Develop:
![Screenshot 2024-12-01 at 15 53 01](https://github.com/user-attachments/assets/7d1218a3-4b81-490a-9fd3-82c4f371bb5d)

Your branch:
<img width="805" alt="Screenshot 2024-12-01 at 15 46 16" src="https://github.com/user-attachments/assets/d5012027-9977-435f-b58a-14b48d26d20f">

Check how the address behaves. There was `...` before

Reply:
This should be fixed as I removed the added styles on BlurWrapper